samples/cpp/benchmark_app/main.cpp

```diff
@@ -172,13 +170,13 @@ int main(int argc, char* argv[]) {
         // -----------------------------------------------------------
         next_step();
 
-        Core ie;
+        ov::runtime::Core ie;
 
         if (FLAGS_d.find("CPU") != std::string::npos && !FLAGS_l.empty()) {
             // CPU (MKLDNN) extensions is loaded as a shared library and passed as a
             // pointer to base extension
             const auto extension_ptr = std::make_shared<InferenceEngine::Extension>(FLAGS_l);
```

Comment:
What if user wants to use new extensions?
Moreover extension can be used only for unsupported op.

Reply:
That's previously existing behavior. Modifying it is out of scope of this PR. Please create FR if it is needed.